website/docs/components/separator/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,44 @@
+## Usage
+
+### When to use
+
+- Use to visually separate different parts or sections on a page.
+- Use it to create a clear break between paragraphs, blocks, or sections.
+
+### When not to use
+
+- Don’t use to separate headings and body text. Instead, use the proper [Display token](https://github.com/hashicorp/design-system/issues/262) to emphasize hierarchy. 
```

Comment:
This may be an outlier component that doesn't benefit from the usage information in this structure as this content feels more appropriate as a part of the "Placement" section. I've provided a few suggestions below about where it could go instead.
```suggestion

```